### DISCUSSION TITLE
A (tentative) guide to build a GitHub portfolio page

### DISCUSSION THREAD

**Discussion by Garcia-INPE** [cu-esiil-edu/2024-data-short-course#27] [Show and tell]
I've compiled some information about the steps to build a GitHub Page from the ESIIL Data Short Course (taught by @nquarder, @eculler and colleagues) and the [instructions from GitHub itself](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site) and the result is [here](https://github.com/Garcia-INPE/Tutorials/blob/71648807f25078afe36a29cd05d111ddf037bcb5/Tut-GitHubPages.md). Questions? Comments? Suggestions? Fixes?

**Comment by oye-bobs** [comment]
This is super helpful @Garcia-INPE .
